src/gui/mainwindow.cpp

```diff
@@ -25,6 +25,9 @@
 
 #include "../intermediate_structures.h"
 #include "../posture_estimator.h"
+#include "settingswindow.h"
+
+// #include "opencv/cv.hpp"
```

Comment:
If we don't need this include could we remove?

Reply:
No problem!